docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,104 @@
+# Tizen 6.0
+
+The second milestone (M2) release of Tizen 6.0 was announced in October 2020.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+The key features of Tizen 6.0 are the enhancement of development environment, core and AI component.
+
+## Enhancement of Development Environment and Core Component
+
+### Tizen.NET
+
+Tizen suppports C# development environment based on .NET Core for application development productivity.
+In Tizen 6.0 the latest .NETCore 3.1 and the latest version of cross-platform Xamarin is supported.
+
+There is a advance in the application start-up time and memory usage reduction.
+and "XAML Hot Reload" beta service is supported as well.
+
+- **.NET Core** :
+Latest .NET Core 3.1(LTS)
+Optimized performance
+Diagnostics tools
+
+- **TizenFX** :
+Tizen platform-specific API set
+API level 8 Released
+
+- **Xamarin** :
+Latest Xamarin.Forms (4.8.0) & Xamarin.Essentials (1.5.3) are available.
+TV UI Controls & CircularUI : New UI controls for Smart TVs and Smart Watches.
+XAML Hot Reload : Boosting Developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device various offloading services
+
+Tizen 6.0 provides the multi-device offloading service using web technology.
+
+The complexity of applications such as A.I and AR/VR increases in web side, maximization of computational performance within web contents is needed.
+
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen java script engine for the product having Hardware resource restriction.
+It shows about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine.
+It supports ES6 full specficiation and supports up to a large part of ES 10.
+
+- **Lightweight JavaScript engine (ESCARGOT)** in https://github.com/Samsung/excargot
+
+  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
+    - Binary size : 13.7 %
+    - Average RSS : 35.1 %
+
+  - Features
+    - Support ES6 Full feature and partial support of ES10
```

Comment:
Escargot is the Tizen JavaScript engine for the product having Hardware resource restriction. Escargot shows about 14% in binary size and 35% in memory usage for the benchmark application compared to Chrome's V8 JavaScript engine. It supports ES6 full specification and supports up to a large part of ES 10.

**Question: 14% increase or decrease 35 %  increase or decrease**


- **Lightweight JavaScript engine (ESCARGOT)** in https://github.com/Samsung/excargot 

Question: Why is **in** used, what exactly you wish to convey)

  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
    - Binary size : 13.7 %
    - Average RSS : 35.1 %

  - Features
    - Support ES6 Full feature and partial support of ES10

Reply:
It means "Escargot use 14% in binary size"  I change the word ( shows --> uses )